README.md

```diff
@@ -2,50 +2,23 @@
 [//]: # "To regenerate it, use `python -m synthtool`."
 <img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
 
-# [The flaky.dev test tracking service: Node.js Client](https://github.com/google/flaky-service)
+# [flaky.dev: a flaky test identification service](https://github.com/google/flaky-service)
 
 [![release level](https://img.shields.io/badge/release%20level-beta-yellow.svg?style=flat)](https://cloud.google.com/terms/launch-stages)
 [![npm version](https://img.shields.io/npm/v/flaky-service.svg)](https://www.npmjs.org/package/flaky-service)
 [![codecov](https://img.shields.io/codecov/c/github/google/flaky-service/master.svg?style=flat)](https://codecov.io/gh/google/flaky-service)
 
 
-
-
-
-
-
-
-
 * [github.com/google/flaky-service](https://github.com/google/flaky-service)
 
-Read more about the client libraries for Cloud APIs, including the older
-Google APIs Client Libraries, in [Client Libraries Explained][explained].
-
-[explained]: https://cloud.google.com/apis/docs/client-libraries-explained
-
-**Table of contents:**
-
-
-* [Quickstart](#quickstart)
-
-  * [Installing the client library](#installing-the-client-library)
-
-
-* [Versioning](#versioning)
-* [Contributing](#contributing)
-* [License](#license)
-
 ## Quickstart
 
-### Installing the client library
-
-```bash
-npm install flaky-service
-```
-
-
-
+## Developing
 
+* `docker-compose.yml`: run `docker-compose up`, to start the flaky.dev service
```

Comment:
Instead of compose, would it be possible to use scaffold and minikube?  These are a little more aligned to our strategy.

Reply:
I threw this in last night so that someone could at least consider running the project locally... I expect we'll throw out postgres for firestore in the first week or two of the project...

Basically as soon as I can get a Pantheon account setup. tldr; I don't know if we should worry too much about Docker related choices right now.